marklogic-data-hub-central/ui/src/components/load/load-card.tsx

```diff
@@ -180,20 +194,27 @@ const LoadCard: React.FC<Props> = (props) => {
                             size="small"
                         >
                             <div className={styles.formatContainer}>
-                                <div style={sourceFormatStyle(elem.sourceFormat)}>{elem.sourceFormat.toUpperCase()}</div>
+                                <div style={sourceFormatStyle(elem.sourceFormat)} aria-label={`${elem.name}-sourceFormat`}>{elem.sourceFormat.toUpperCase()}</div>
                             </div>
-                            <div className={styles.stepNameStyle}>{getInitialChars(elem.name, 25, '...')}</div>
+                            <div className={styles.stepNameStyle} aria-label={`${elem.name}-stepName`}>{getInitialChars(elem.name, 25, '...')}</div>
```

Comment:
Are we adding a label here purely for testing? We dont allow duplicate step names. So do we need a lebel text.

Reply:
Yes, it was added only for testing purpose initially. But i don't think we need it anymore so have removed it.